requirements.txt

```
@@ -1,4 +1,4 @@
 -r _building/requirements.txt
 
 
-k3ut~=0.1.7
+k3ut~=0.1.15
```

```suggestion
k3ut>=0.1.15,<0.2
```

With `~=`, a minor version change of `k3ut` results in a version conflict, if two package depends on different versions of `k3ut`.

`~=0.x` is compatible with `0.x.1`, `0.x.2`. While `~=0.x.y` is compatible only with `0.x.y`.

---

all down~